examples/next-app/locales/client.ts

```diff
@@ -4,14 +4,8 @@ import { createI18nClient } from 'next-international/client';
 export const { useI18n, useScopedI18n, I18nProviderClient, useChangeLocale, defineLocale, useCurrentLocale } =
   createI18nClient(
     {
-      en: async () => {
-        await new Promise(resolve => setTimeout(resolve, 100));
-        return import('./en');
-      },
-      fr: async () => {
-        await new Promise(resolve => setTimeout(resolve, 100));
-        return import('./fr');
-      },
+      en: () => import('./en'),
+      fr: () => import('./fr'),
```

Comment:
Can we keep this artificial delay? It's helpful to make sure we don't have any layout shifts / unwanted loaded when changing locale.

Reply:
I've done a few tests and as the `changeLocale` function does the following : import the translations from the next locale, then push the new locale to the URL and refresh the page

I don't think we'll have any layout shifts because the result of `import` is cached so it'll be reused.

I've made a video to compare with and without the artificial delay

https://github.com/QuiiBz/next-international/assets/31007601/f28132ff-201b-4257-8ad8-8d5425f2b9a1

Reply:
This artificial delay allows us to see that the suspense boundary is triggered when we switch to another locale, which we absolutely don't want to happen, and should also prevent https://github.com/QuiiBz/next-international/issues/208.

Reply:
oh ok, I re-added the promises in `() => import`.
I didn't know about this issue, sorry